ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -0,0 +1,394 @@
+
+$$x := \max\left(x-\frac{s}{p}, 0\right)$$
+
+and the gas price is updated before the next chunk.
+With the current $p = 2$, half of all gas consumed is added to excess while half of all surplus is deducted.
+
+> [!NOTE]
+> The ACP-103 concept of a leaky gas bucket with capacity $C$ is an emergent phenomenon in SAE.
+
+### JSON RPC methods
+
+Although asynchronous execution decouples the transactions and receipts recorded by a specific block, APIs MUST NOT alter their behavior to mirror this.
+In particular, the API method `eth_getBlockReceipts` MUST return the receipts corresponding to the block's transactions, not the receipts settled in the block.
+
+To achieve this, we first define the _executed_ and _settled_ states for blocks.
+A block is initially _accepted_ by consensus, enqueuing its tranche of transactions.
+Once the last transaction in the tranche has been executed, the block itself is considered to be executed.
+When the same transaction is settled then so too is the block considered settled.
+
+#### Named blocks
+
+The Ethereum Mainnet APIs allow for retrieving blocks by named parameters that the API server resolves based on their consensus mechanism.
+Other than the _earliest_ (genesis) named block, which MUST be interpreted in the same manner, all other named blocks are mapped to SAE in terms of the execution status of all block transactions and MUST be interpreted as follows:
+
+ * _pending_: the most recently _accepted_ block;
+ * _latest_: the block that was most recently _executed_;
+ * _safe_ and _finalized_: the block that was most recently _settled_.
+
+> [!NOTE]
+> The finality guarantees of Snowman consensus remove any distinction between _safe_ and _finalized_. 
+> Furthermore, the _latest_ block is not at risk of re-org, only of a negligible risk of data corruption local to the API node.
+
+
+### Optimal block building
+
+As EOA-to-EOA transfers of value are entirely guaranteed upon _acceptance_, block builders SHOULD prioritise other transactions for earlier execution.
+A reliable marker of such transactions is a gas limit of 21,000 as this is an indication from the sender that they do not intend to execute bytecode.
+
+Checking for code at the destination address is both unnecessary and unreliable as the address might be that of a contract created earlier in the queue.
+
+## Backwards Compatibility
+
+All ACPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The ACP should provide recommendations for dealing with these incompatibilities.
+
+## Reference Implementation
+
+A reference/example implementation that people can use to assist in understanding or implementing this specification. If the implementation is too large to reasonably be included inline, then consider adding it to the ACP subdirectory or linking to a PR in an external repository.
+
+## Security Considerations
```

Comment:
I think everything currently in the Security Considerations section needs to be moved into the Specification section.

The Security Considerations section should talk about the tradeoffs made (including the calculations of the worst case DoS multiplier and argument that the multiplier on the fee isn't that bad). The Specification section should describe what verification needs to take place (so that the DoS multiplier is bounded).